doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -351,19 +353,18 @@ Follow these instructions to build the RT kernel.
 
    .. code-block:: none
 
+      $ cp x86-64_defconfig .config
       $ make olddefconfig
       $ make targz-pkg
 
 #. Copy the kernel and modules:
 
    .. code-block:: none
 
-      $ sudo mount /dev/sda1 /mnt
-      $ sudo cp arch/x86/boot/bzImage /mnt/EFI/
-      $ sudo umount /mnt
       $ sudo mount /dev/sda2 /mnt
-      $ sudo cp linux-4.19.72-rt25-x86.tar.gz -P /mnt/usr/lib/modules/ && cd /mnt/usr/lib/modules/
-      $ sudo tar zxvf linux-4.19.72-rt25-x86.tar.gz
+      $ sudo cp arch/x86/boot/bzImage /mnt/boot/
+      $ sudo tar -zxvf linux-4.19.72-rt25-x86.tar.gz -C /mnt/lib/modules/
+      $ sudo cp -r /mnt/lib/modules/lib/modules/4.19.72-rt25 /mnt/lib/modules/
```

Comment:
Why??

Reply:
linux-4.19.72-rt25-x86.tar.gz includes some folders, like lib/modules/4.19.72-rt25 and boot.

Reply:
Could we do this like this then?
```
$ sudo tar -zxvf linux-4.19.72-rt25-x86.tar.gz -C /mnt/
```

Reply:
yes, I think we can decompress the "tar"packet anywhere, only need copy the related folder into the right folder.

Reply:
Ah, I got your point, that's more easy and directly.